File: tasks/translation/output/Qwen/Qwen2.5-32B-Instruct-AWQ/codenet/cot/Java/Go/s063482767_commented.go
package main

import (
	"fmt"
	"sort"
)

func main() {
	var n int
	fmt.Scan(&n)
	queue := make([]int, n)
	for i := 0; i < n; i++ {
		fmt.Scan(&queue[i])
	}
	sort.Ints(queue)
	sum := 0
	for i := 0; i < n-1; i++ {
		sum += queue[i]
	}
	if queue[n-1] < sum {
		fmt.Println("Yes")
	} else {
		fmt.Println("No")
	}
	// End of Code
}

File: tasks/translation/output/Qwen/Qwen2.5-32B-Instruct-AWQ/codenet/direct/comment/uncomment/direct/Java/Python/s063482767_commented.py
import heapq

def main():
    # write your code here
    sc = [int(x) for x in input().split()]
    n = sc[0]
    queue = []
    for i in range(n):
        queue.append(sc[i+1])
    heapq.heapify(queue)
    
    sum = 0
    for _ in range(n - 1):
        sum += heapq.heappop(queue)

    # 出力
    print("Yes" if heapq.heappop(queue) < sum else "No")

main()  # End of Code
